api/src/backend/api/migrations/0006_findings_first_seen.py

```diff
@@ -0,0 +1,58 @@
+from django.db import migrations, models
+
+
+def populate_first_seen(apps, schema_editor):
+    # First, update findings where delta='new'
+    schema_editor.execute(
+        """
+        UPDATE findings
+        SET first_seen = inserted_at
+        WHERE delta = 'new';
+    """
+    )
+
+    # Then update all other findings
+    schema_editor.execute(
+        """
+        UPDATE findings f1
+        SET first_seen = (
+            SELECT MIN(f2.inserted_at)
+            FROM findings f2
+            WHERE f2.uid = f1.uid
+            AND f2.delta = 'new'
+        )
```

Comment:
I don't know if the following query will be simpler and more performant but we only need to get the first one ordering by `finding.id` and limiting the result to 1, because as is using UUIDv7 the timestamp is present, so with that you wouldn't need the `MIN()` to get it.